classes/class-loader.php

```diff
@@ -463,7 +464,9 @@ public function enqueue_campaigns_assets() {
 
 		$campaign_theme = $post->theme ?? $post->custom['_campaign_page_template'] ?? null;
 
-		if ( ! is_string( $campaign_theme ) || empty( $campaign_theme ) ) {
+		if ( ! is_string( $campaign_theme ) || empty( $campaign_theme )
+			|| ! in_array( $campaign_theme, PostCampaign::LEGACY_THEMES, true )
```

Comment:
This is the part where it enqueues the files with CSS overrides. They only exist for the legacy themes.